bookkeeper-proto/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -63,6 +63,7 @@ enum OperationType {
     READ_LAC = 7;
     GET_BOOKIE_INFO = 8;
     START_TLS = 9;
+    FORCE_LEDGER = 10;
```

Comment:
I thought we are using SYNC_LEDGER instead of FORCE. Force doesn't tell what it is forcing. I was feeling weird in the previous patch itself but did not click on what it was I was missing in the name. :) s/force/sync makes lot of sense and code readability IMO